### ISSUE TITLE
Mark `resolveResource` as an activation event

### ISSUE THREAD

**Issue by nturinski** [microsoft/vscode-azureresourcegroups#361]
It's artificially inflating our user numbers since we're not excluding it as an activation event, but is happening pretty frequently because of the new view.  I tried filtering this event out, and the numbers look normal.

**Comment by alexweininger** [comment]
We'll have to do this in each extension.

**Comment by nturinski** [comment]
Yeah, we can edit the query in our dashboards, but since partner teams are using our telemetry, we definitely need to fix that 😐

**Comment by nturinski** [comment]
We _also_ have to mark `provideResources` as an activation event as it triggers as soon as someone looks at the Local Workspace view. If we have a similar view for Actions where extensions contribute, that'll be another place to do so.

I believe only Functions, Storage, and Databases provide local resources atm
